README.md

```diff
@@ -49,13 +43,12 @@ Configuring Your AndroidManifest
     </application>
     ```
 
--   Enable the `INTERNET`, `ACCESS_NETWORK_STATE` and `GET_TASKS` permissions:
+-   Optional: Enable the `ACCESS_NETWORK_STATE` and `GET_TASKS` permissions:
```

Comment:
@loopj What is your opinion about this comment by @JakeWharton : https://github.com/bugsnag/bugsnag-android/pull/91#discussion_r48775577